shared/common-adapters/profile-card.tsx

```diff
@@ -138,13 +178,22 @@ const ProfileCard = ({clickToProfile, showClose, containerStyle, username}: Prop
         <>
           <ServiceIcons userDetails={userDetails} />
           {!!userDetails.bio && (
-            <Kb.Text type="Body" center={true}>
-              {userDetails.bio}
+            <Kb.Text type="Body" center={true} lineClamp={4} ellipsizeMode="tail">
+              {userDetails.bio.replace(/\s/g, ' ')}
             </Kb.Text>
           )}
         </>
       )}
-      {!isSelf &&
+      {clickToProfile && (
+        <Kb.Button
+          style={styles.button}
+          type="Default"
+          mode="Secondary"
+          label="View profile"
+          onClick={openProfile}
+        />
+      )}
+      {showFollowButton &&
```

Comment:
It might be a good idea to have the follow button always appear on top of the view profile button so the follow action doesn't unexpectedly replace the view profile button: 
![repro](https://user-images.githubusercontent.com/54032873/66775167-672a3a00-ee91-11e9-8b40-48c1c7c0f345.gif)

Reply:
Good point; will do!